deploy/monitor/operator_service.yaml

```diff
@@ -0,0 +1,17 @@
+# Monitor Service (Metrics)
```

Comment:
I think the operator itself creates this Service, see: https://github.com/aerogear/unifiedpush-operator/blob/master/cmd/manager/main.go#L131..L135

Reply:
It is true @grdryn. @austincunningham why you create it in the MSS Oper since there are working equals. see here https://github.com/aerogear/mobile-security-service-operator/blob/master/cmd/manager/main.go#L102? Why it was required?

Reply:
@camilamacedo86 I wasn't able to see metrics outside of the operator until I added the service. But it may have been coincidental , Easiest way to check is to deploy it on a integr8ly cluster RHPDS and remove the service

Reply:
I will add a todo check for we see it. Tks for your input @austincunningham

Reply:
I think that remove it will not work with integr8ly since the default created has no the label as follows.

```go
  labels:
    monitoring-key: middleware
```